docs/styles.md

```diff
@@ -0,0 +1,34 @@
+# Styling content for RAISE
+
+In order to maintain consistency and reliability across all RAISE content, the styling associated with RAISE content is consolidated into a series of css classes located in `src/styles`. 
+
+RAISE Style classes are either written for specific elements, or can be applied to a variety of objects. Those differences are listed below.
+
+## Style Classes 
+
+### Indent (os-raise-indent)
+Add a padding to the left side of an element. If the element already includes a left padding (like an ordered list) the indent will get set to exactly 32px
+**Example**
+
+**Availiability**
+Any element
+**Usage**
```

Comment:
I think you need an empty line before this (it currently shows up on the same line as the preceding sentence).